src/Data/Currency.purs

```diff
@@ -23,20 +26,48 @@ derive instance newtypeCents :: Newtype Cents _
 derive newtype instance eqCents :: Eq Cents
 
 instance encodeJsonCents :: EncodeJson Cents where
-  encodeJson = encodeJson <<< BigInt.toNumber <<< unwrap
+  encodeJson = encodeJson <<< centsToNumber
+
+instance decodeJsonCents :: DecodeJson Cents where
+  decodeJson json = do
+    num <- decodeJson json
+    note ("Could not convert value to `Cents`: " <> show num) (parseCentsFromNumber num)
 
 instance showCents :: Show Cents where
-  show (Cents n) = "Cents " <> BigInt.toString n
+  show (Cents n) = "(Cents " <> BigInt.toString n <> ")"
+
+----------
+-- FROM CENTS
+
+centsToMaybeInt :: Cents -> Maybe Int
+centsToMaybeInt = Int.fromString <<< BigInt.toString <<< unwrap
+
+centsToNumber :: Cents -> Number
+centsToNumber = BigInt.toNumber <<< unwrap
+
+centsToString :: Cents -> String
+centsToString = BigInt.toString <<< unwrap
+
+----------
+-- TO CENTS
+
+centsFromInt :: Int -> Cents
+centsFromInt = Cents <<< BigInt.fromInt
 
--- | Will parse cents from a 32bit int
 parseCentsFromNumber :: Number -> Maybe Cents
 parseCentsFromNumber = map Cents <<< BigInt.fromNumber
 
-parseCentsFromMicroDollars :: Number -> Maybe Cents
-parseCentsFromMicroDollars = parseCentsFromNumber <<< flip (/) 10000.0
+parseCentsFromString :: String -> Maybe Cents
+parseCentsFromString = map Cents <<< BigInt.fromString
 
-centsToMaybeInt :: Cents -> Maybe Int
-centsToMaybeInt = Int.fromString <<< BigInt.toString <<< unwrap
+parseCentsFromMicroDollars
+  :: Warn (Text "parseCentsFromMicroDollars is deprecated and will be removed in a future release")
```

Comment:
Ocelot only deals with Currency/Cents, Microdollars are specific to Twitter's domain, so dealing with them should just be done in application code. 

I didn't want to force the change right away in Wildcat though, so I figured a deprecation warning was suitable.